### PR TITLE
Enable the removal of latest stable version.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: app/bin/tools/remove_package.dart [hunk 1]
```diff
@@ -161,13 +161,28 @@ Future removePackageVersion(String packageName, String version) async {
...
+          'Last version detected. Use full package removal without the version qualifier.');
+    }
+
+    bool updatePackage = false;
+    if (package.latestVersion == version) {
+      package.latestVersionKey = null;
+      updatePackage = true;
+    }
+    if (package.latestDevVersion == version) {
+      package.latestDevVersionKey = null;
```
Comment (jonasfj) [inline app/bin/tools/remove_package.dart]: Don't we need to set this to something?

Comment (isoos) [reply app/bin/tools/remove_package.dart]: Yeah, it looks like we always populate this, even if a package has no dev version. I'd keep it this way now, and we can do a cleanup of that later.

**Comment by isoos** [comment]
Tested it on staging (with the NPE fix).

File: app/bin/tools/remove_package.dart [hunk 2]
```diff
@@ -161,13 +161,30 @@ Future removePackageVersion(String packageName, String version) async {
...
-    if ('${package.latestSemanticVersion}' == version) {
-      throw Exception('Cannot delete the latest version of $packageName.');
+    if (versionNames.length == 1 && versionNames.single == version) {
+      throw Exception(
+          'Last version detected. Use full package removal without the version qualifier.');
+    }
+
+    bool updatePackage = false;
+    if (package.latestVersion == version) {
+      package.latestVersionKey = null;
```
Comment (jonasfj) [inline app/bin/tools/remove_package.dart]: Don't we need to update it to something other than `null`

File: app/lib/frontend/models.dart [hunk 3]
```diff
@@ -101,14 +101,14 @@ class Package extends db.ExpandoModel {
 
   void updateVersion(PackageVersion pv) {
```
Comment (jonasfj) [inline app/lib/frontend/models.dart]: Should we document what this does... just a one line comment..

**Pull request merged**